integration-cli/docker_cli_stat_test.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"os/exec"
+	"testing"
+	"time"
+)
+
+func TestCliStatsNoStream(t *testing.T) {
+	defer deleteAllContainers()
+	var (
+		name   = "statscontainer"
+		runCmd = exec.Command(dockerBinary, "run", "-d", "--name", name, "busybox", "top")
+	)
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		t.Fatalf("Error on container creation: %v, output: %v", err, out)
+	}
+
+	chErr := make(chan error)
+	go func() {
+		chErr <- exec.Command(dockerBinary, "stats", "--no-stream", name).Run()
+	}()
+
+	select {
+	case err := <-chErr:
+		if err != nil {
+			t.Fatalf("Error running stats: %v", err)
+		}
+	case <-time.After(2 * time.Second):
```

Comment:
Is 2 seconds enough for stats to return? Is there any extreme situation that will break this?
maybe 3 seconds is much safer :)